pkg/gitqlite/display.go

```diff
@@ -126,8 +161,9 @@ func tableDisplay(rows *sql.Rows, write io.Writer) error {
 	}
 	table := tablewriter.NewWriter(write)
 	table.SetHeader(columns)
+	numRows := 0
 	for rows.Next() {
-
+		numRows++
```

Comment:
curious about `numRows` here - is it used anywhere? possibly holdover from testing/outputting something different?

Reply:
that is a holdover. I was initially thinking of having it be an automatic process but then realized that would involve creating a counter in each of the outputs which would then be an issue if the format had already been specified. Definitely can rm this